dataclients/kubernetes/testdata/ingressV1/eastwestrange/ewring-without-internal-host-1prule.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: qux
+  namespace: foo
+spec:
+  rules:
+  - host: www.example.org
+    http:
+      paths:
+      - path: "/"
```

Comment:
I think this definition is not valid
https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122
> pathType is now required for each specified path.

Reply:
A lot of v1 definitions are not valid. I am wondering if we could either validate them automatically or convert beta versions to v1 using https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-kubectl-convert-plugin maybe

Reply:
There are also not test fixtures for pathType `Prefix`, `Exact` and `ImplementationSpecific`

Reply:
fixed and also added separate tests for `Prefix` and `Exact` and both with `ImplementationSpecific` in one ingress definition.